### DISCUSSION TITLE
Where can I find the api documentation for addon-actions?

### DISCUSSION THREAD

**Discussion by TheMikeyRoss** [storybookjs/storybook#24421] [Documentation] [answered]
I'm trying to use `@storybook/addon-actions` and I can't find the api documentation online. 

For example I want to learn more about this `action` method
![image](https://github.com/storybookjs/storybook/assets/97709651/3ce47320-84c9-42c3-ad24-66d3fd2e3e77)

**Comment by zaaakher** [comment, marked as answer]
You can find more information about `addon-actions` in [here](https://github.com/storybookjs/storybook/blob/master/addons/actions/ADVANCED.md). Although it's not recommended to use that addon and instead you can use the new approach documented [here](https://storybook.js.org/docs/react/essentials/actions)

```js
argTypes : {
   handleRouteToRegister: {
      action:" CustomName"
   }
}
```
